specs/window_functions.toml

```diff
@@ -0,0 +1,33 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
+statements = [
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00001*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00002*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00003*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00004*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00005*' with (compression = 'gzip')",
+    "refresh table uservisits"
+]
+
+[[queries]]
+statement = '''select avg("adRevenue") over (order by "cCode")
+                from uservisits'''
+iterations = 50
+min_version = '3.2.0'
+
+[[queries]]
+statement = '''select last_value("adRevenue") over (order by "cCode"),
+                avg("duration") over(order by "sourceIP")
+                from uservisits'''
+iterations = 50
+min_version = '3.3.0'
+
+[[queries]]
+statement = '''select nth_value("adRevenue", 3) over (order by "cCode")
```

Comment:
I think it would make sense to use the same query as one of the above here. So that we've a direct comparison: Same query with different concurrency). Otherwise we mix 2 variables and it becomes more difficult to get any meaningful insights.

Reply:
that's a good point, thanks for pointing out. I'm adding a new one though as the one above is a use case of having different windows